chisel/README.md

```diff
@@ -0,0 +1,98 @@
+# `chisel`
+
+Chisel is a fast, utilitarian, and verbose solidity REPL.
+
+## Why?
+
+Ever wanted to quickly test a small feature in solidity?
+
+Perhaps to test how custom errors work, or how to write inline assembly?
+
+Chisel is your solution. Chisel let's you write, execute, and debug Solidity directly in the command line.
+
+Once you finish testing, Chisel even lets you export your code to a new solidity file!
+
+In this sense, Chisel even serves as a project generator.
+
+
+
+## Checklist
+
+- [ ] REPL functionality
+  - [ ] Create temporary REPL contract (in memory, or temp file?).
+    - [ ] Implement `forge-std/Test.sol` so that cheatcodes etc. can be used.
+  - [ ] Utilize foundry's `evm` module for REPL env.
+    - [ ] Implement network forking.
+  - [ ] Import all project files / interfaces into REPL contract automatically.
+    - [ ] Optionally disable this functionality with a flag.
```

Comment:
Do `foundry.toml` files support an `[anvil]` section? I don't think they do, but it might be worth adding `[anvil]` and `[chisel]` section support. 

The reason I thought of this is because sometimes if you import all files/interfaces you'll get solc build errors, e.g. if two deps have different IERC20 files and now our repl imports them both, then you get a duplicate declaration error. So one idea is making this auto-import functionality (which is awesome) granular so you can use the `(no-)match-*` family of flags to choose what does/doesn't get imported by default. Can definitely support that as a CLI arg too of course